### PR TITLE
docs(static): add plugin submission how-to

### PR DESCRIPTION
**Pull request by tolzhabayev**
**What this PR does / why we need it**:
The current documentation states that in order to submit a plugin one has to open a PR in `grafana-plugins-repository`.
This PR updates the docs to reflect the current process of submission via grafana cloud account.

The referenced static images is [here](https://github.com/grafana/website/pull/7255).

### CODE REVIEW

File: docs/sources/developers/plugins/package-a-plugin.md [hunk 1]
```diff
@@ -44,4 +44,58 @@ There are three steps needed to package a plugin:
...
 The best way to share your plugin with the world is to publish it on [Grafana Plugins](https://grafana.com/plugins). By having your plugin published on Grafana.com, more users will be able to discover your plugin.
 
-To publish a plugin to [Grafana Plugins](https://grafana.com/grafana/plugins), create a pull request to the [Grafana Plugin Repository](https://github.com/grafana/grafana-plugin-repository). Please note that both the source code and the packaged plugin archive need to be publicly available.
+Before you submit your plugin, we ask that you read our guidelines and frequently asked questions.
+
+### Guidelines
+
+To speed up the time it takes to review your plugin:
+
+- Check that your plugin is ready for review using the plugin validator.
```
Comment (marcusolsson) [inline docs/sources/developers/plugins/package-a-plugin.md:53]:
```suggestion
- Check that your plugin is ready for review using the [plugin validator](https://github.com/grafana/plugin-validator).
```

File: docs/sources/developers/plugins/package-a-plugin.md [hunk 2]
```diff
@@ -44,4 +44,58 @@ There are three steps needed to package a plugin:
...
+- Read our 6 tips for improving your Grafana plugin before you publish
```
Comment (marcusolsson) [inline docs/sources/developers/plugins/package-a-plugin.md:54]:
```suggestion
- Read our [6 tips for improving your Grafana plugin before you publish](https://grafana.com/blog/2021/01/21/6-tips-for-improving-your-grafana-plugin-before-you-publish/).
```

**Review by marcusolsson — APPROVED** [review]

File: docs/sources/developers/plugins/package-a-plugin.md [hunk 3]
```diff
@@ -44,4 +44,58 @@ There are three steps needed to package a plugin:
...
+
+**Do I need to submit a private plugin?**
+
+- No. Please only submit plugins that you wish to make publicly available for the Grafana community.
+
+**How long does it take to review my submission?**
+
+- We're not able to give an estimate at this time, though we're constantly working on improving the time it takes to review a plugin.
+
+## Publishing your plugin for the first time
```
Comment (leventebalogh) [inline docs/sources/developers/plugins/package-a-plugin.md:66]: **Question:** is H2 `##` intended here?

Probably it is, only asking it to be sure.

**Review by leventebalogh — APPROVED** [review]
Nice 👏

File: docs/sources/developers/plugins/package-a-plugin.md (hunk 3 above)
Comment (tolzhabayev) [reply docs/sources/developers/plugins/package-a-plugin.md:66]: @leventebalogh good catch, in this place it is intended indeed. But to make the structure more logical I have changed the previous h2 `Publish your plugin on Grafana.com` to h1.

**Pull request merged**